R/aggregate.R

```diff
@@ -0,0 +1,166 @@
+##' This function calculates the robust summarisation for each feature
+##' (protein). Note that the function assumes that the intensities in
+##' input `e` are already log-transformed.
+##'
+##' @title Return the Robust Expression Summary of a matrix
+##' @param x A feature by sample `matrix` containing quantitative
+##'     data with mandatory `colnames` and `rownames`.
+##' @param ... Additional arguments passed to [MASS::rlm()].
+##' @return `numeric()` vector of length `ncol(x)` with robust
+##'     summarised values.
+##' 
+##' @author Adriaan Sticker, Sebastian Gibb and Laurent Gatto
```

Comment:
Shouldn't Adriaan Sticker also be added as contributor to the `Authors@R` or at least to the `README.md`?

Reply:
Done

Reply:
Ok, until now I also added the contributors to the `README.md` where I briefly described what they did.

Reply:
Added Adriaan to the README.